go/src/meguca/geoip/db.go

```diff
@@ -4,33 +4,57 @@ import (
 	"net"
 	"strings"
 	"time"
+	"os"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"crypto/md5"
+	"path/filepath"
+
+	"meguca/db"
 
 	"github.com/oschwald/maxminddb-golang"
 	"github.com/go-playground/log"
+	"github.com/mholt/archiver"
 )
 
 // nil, if database not loaded
-var db *maxminddb.Reader
+var gdb *maxminddb.Reader
```

Comment:
Needs mutex protection now that the db could be loaded after server start.

Reply:
I feel stupid now.
I did that when I was tired, my bad.